### DISCUSSION TITLE
Connection reset by peer

### DISCUSSION THREAD

**Discussion by JoseffYu** [community/community#70821] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I wanna push folder into my repo, I always meet this problem:fatal: unable to access '/my/repo': Recv failure: Connection reset by peer

sometimes when I want to clone files, this problem also occurs

**Comment by JoseffYu** [comment]
I know there is already a great answer to this problem, but that cannot solve my problem properly. I can still meet this problem

**Reply by k8ieone** [reply]
Are you by any chance in Europe?

**Reply by JoseffYu** [reply]
im currently in UK🥲

**Reply by k8ieone** [reply]
Interesting, the issue disappeared later today

**Reply by JoseffYu** [reply]
cool! I just received the email from github support saying this issue resolved

**Comment by k8ieone** [comment, marked as answer]
I ran into this today, I believe this to be an issue on the side of Github.

I ran a script that kept pulling one of my repos and every once in a while, the connection reset would happen.

I did the same with a GitLab project and I couldn't get it to happen, not even once.
